cni-plugin/tests/calico_cni_k8s_test.go

```diff
@@ -721,6 +721,52 @@ var _ = Describe("Kubernetes CNI tests", func() {
 				numIPv4IPs:      1,
 				numIPv6IPs:      1,
 			},
+			{
```

Comment:
This scenario is failing because the node created as part of this set of tests doesn't have dual-stack CIDRs present: https://github.com/projectcalico/calico/blob/master/cni-plugin/tests/calico_cni_k8s_test.go#L859-L865

We probably want a new section for dual-stack tests so that we don't reduce coverage for the legacy / single-stack case. i.e., we want tests for both `PodCIDR` and `PodCIDRs`

Reply:
I tried something :)